dashboard/v1.1/.eslintrc.json

```diff
@@ -0,0 +1,64 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "extends": ["airbnb"],
```

Comment:
can we have a rule for `2 space` shift?

Reply:
yarn seems okay to me!

Reply:
I dont remember using yarn anywhere. Create-react-app may have created it. I am not sur though @Harkishen-Singh